### PR TITLE
Integrate arbitrary compliance

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Custom requirements](https://trello.com/c/U52YgpVe/817-custom-requirements)

File: new-lamassu-admin/src/pages/Triggers/CustomInfoRequests/CustomInfoRequests.js [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+      setHasError(false)
+      setToBeEdited(null)
+      toggleWizard()
+    },
+    refetchQueries: () => ['customInfoRequests']
+  })
+
+  const [removeEntry] = useMutation(REMOVE_ROW, {
+    onError: () => {
+      console.log('Error while removing custom info request')
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Triggers/CustomInfoRequests/CustomInfoRequests.js:101]: These are not handy other than debugging, right?

File: new-lamassu-admin/src/pages/Triggers/CustomInfoRequests/CustomInfoRequests.js [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    onCompleted: () => {
+      setHasError(false)
+      toggleWizard()
+    },
+    refetchQueries: () => ['customInfoRequests']
+  })
+
+  const [editEntry] = useMutation(EDIT_ROW, {
+    onError: () => {
+      console.log('Error while editing custom info request')
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Triggers/CustomInfoRequests/CustomInfoRequests.js:88]: same

File: new-lamassu-admin/src/pages/Triggers/CustomInfoRequests/CustomInfoRequests.js [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+  const classes = useStyles()
+
+  const [toBeDeleted, setToBeDeleted] = useState()
+  const [toBeEdited, setToBeEdited] = useState()
+  const [deleteDialog, setDeleteDialog] = useState(false)
+  const [hasError, setHasError] = useState(false)
+
+  const [addEntry] = useMutation(ADD_ROW, {
+    onError: () => {
+      console.log('Error while adding custom info request')
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Triggers/CustomInfoRequests/CustomInfoRequests.js:76]: same

**Review by josepfo — CHANGES_REQUESTED** [review]

**Review by josepfo — APPROVED** [review]

**Pull request merged**